src/pseudopeople/entity_types.py

```diff
@@ -26,7 +26,7 @@ class RowNoiseType:
 
     name: str
     noise_function: Callable[[str, pd.DataFrame, ConfigTree, RandomnessStream], pd.DataFrame]
-    row_probability: float = 0.0
+    row_probability: Union[Dict[str, float], float, int] = 0.0
```

Comment:
Can it ever actually be an int? I guess 0 or 1? I've been trying to stay away from ints b/c they can cause so many issues.

Reply:
Well I was copying what we have for cell probability for column noise and for cell probability we have Optional[float] = 0.01 so that also needs to be changed. Im not sure if 0 or 1 would cause and issue but all our error messages say ints or floats so I can verify it works.

Reply:
Floats do work so I'm going to leave it.